lambdas/data-migration2/tests/test-executions.js

```diff
@@ -224,12 +224,40 @@ test.serial('migrateExecutionRecord throws RecordAlreadyMigrated error for alrea
   const newExecution = fakeExecutionFactoryV2({ parentArn: undefined });
 
   await migrateExecutionRecord(newExecution, t.context.knex);
+
+  const olderExecution = {
+    ...newExecution,
+    updatedAt: Date.now() - 1000,
+  };
+
   await t.throwsAsync(
-    migrateExecutionRecord(newExecution, t.context.knex),
+    migrateExecutionRecord(olderExecution, t.context.knex),
     { instanceOf: RecordAlreadyMigrated }
   );
 });
 
+test.serial('migrateExecutionRecord updates an already migrated record if the updated date is newer', async (t) => {
+  const fakeExecution = fakeExecutionFactoryV2({
+    parentArn: undefined,
+    updatedAt: Date.now() - 1000,
+  });
+  await migrateExecutionRecord(fakeExecution, t.context.knex);
+
+  const newerFakeExecution = {
+    ...fakeExecution,
+    updatedAt: Date.now(),
+  };
+  await migrateExecutionRecord(newerFakeExecution, t.context.knex);
+
+  const createdRecord = await t.context.knex.queryBuilder()
```

Comment:
Can we update this to use the execution PG model `.get`?

Reply:
https://github.com/nasa/cumulus/pull/2162/commits/7027bf1243c36dac3abc1052c1e7b28bdd60a305